balena/models/tag.py

```diff
@@ -148,14 +148,9 @@ def get_all_by_device(self, uuid):
 
         """
 
-        device = self.device.get(uuid)
-
-        params = {
-            'filter': 'device',
-            'eq': device['id']
-        }
+        raw_query = '$filter=device/any(d:d/uuid%20eq%20{uuid})'.format(uuid=uuid)
```

Comment:
@Page- isnt it missing a quote? e.g.:
```suggestion
raw_query = "$filter=device/any(d:d/uuid%20eq%20'{uuid}')".format(uuid=uuid)
```

Reply:
Yup: When I execute this code it throws: 
```
Traceback (most recent call last):
  File "/home/otavio/git/balena-sdk-python/test.py", line 6, in <module>
    print(b.models.tag.device.get_all_by_device('01ee734daed728a63186229c33ed606b'))
  File "/home/otavio/git/balena-sdk-python/balena/models/tag.py", line 153, in get_all_by_device
    return super(DeviceTag, self).get_all(raw_query=raw_query)
  File "/home/otavio/git/balena-sdk-python/balena/models/tag.py", line 35, in get_all
    return self.base_request.request(
  File "/home/otavio/git/balena-sdk-python/balena/base_request.py", line 197, in request
    raise exceptions.RequestError(response._content)
balena.exceptions.RequestError: b'"Malformed url: \'/v6/device_tag?$filter=device/any(d:d/uuid%20eq%2001ee734daed728a63186229c33ed606b)\'"'
```

But my suggestion fixes it

Reply:
@otaviojacobi what python version are you using? B/c this seems to be passing CI tests w/ 3.10.

Reply:
Oh, I totally see that we do indeed need the quotes... I guess we are missing a test case for this :disappointed: